test/state/rlp.hpp

```diff
@@ -119,4 +121,116 @@ inline bytes internal::encode_container(InputIterator begin, InputIterator end)
         content += encode(*it);
     return wrap_list(content);
 }
+
+// Load unsigned integral from bytes_view. The destination size must not me smaller than input data.
```

Comment:
```suggestion
// Load unsigned integral from bytes_view. The destination size must not be smaller than input data.
```
Put this comment above `load()`

Reply:
done